### ISSUE TITLE
Extension reports "missing H1 heading" twice

### ISSUE THREAD

**Issue by balajialg** [berkeley-dsep-infra/jupyterlab-a11y-checker#25]
When I launch Data100's Lab 2 notebook using the [nbgitpuller link](https://a11y.datahub.berkeley.edu/hub/user-redirect/git-pull?repo=https%3A%2F%2Fgithub.com%2FDS-100%2Fsp25-student&branch=main&urlpath=lab%2Ftree%2Fsp25-student%2Fhw%2Fhw02B%2Fhw02B.ipynb) in a11y hub, the extension reports "Missing H1 Heading" issue twice. We might need to fix this issue as soon as we can considering the redundant input. Sharing the snapshot below for reference

<img width="973" height="588" alt="Image" src="https://github.com/user-attachments/assets/f881fa29-da8b-4ad2-9658-487c4b1ac3c2" />

Another example from Data100 Lab 11 Notebook:

<img width="973" height="588" alt="Image" src="https://github.com/user-attachments/assets/a11a560e-1429-46fb-9e94-9725e6336c58" />

Here is the [nbgitpuller link](https://a11y.datahub.berkeley.edu/hub/user-redirect/git-pull?repo=https%3A%2F%2Fgithub.com%2FDS-100%2Fsp25-student&branch=main&urlpath=lab%2Ftree%2Fsp25-student%2Flab%2Flab11%2Flab11.ipynb) to launch this notebook.

**Comment by Chanbinski** [comment]
Issue fixed by simplifying the logic. Now we only flag Missing H1 Heading only one per notebook, when a H1 tag is not present at the very top of the notebook. Check the latest version and see if it resolves, and close the issue!

**Comment by balajialg** [comment]
@Chanbinski Thanks! I will test the extension and close the issue after testing few examples

**Comment by balajialg** [comment]
I can confirm that the "Missing H1 headers" issue was reported twice while using version 0.2.2, and was reported once when I used the latest version of a11y-checker (v0.2.5) to test Data100's lab 2 and 11 notebooks. Closing this issue since it is resolved.
